src/standalone/Bucket/Bucket.jl

```diff
@@ -101,24 +101,31 @@ end
 
 """
     BulkAlbedoStatic{FT}(
-        regrid_dirpath::String;
+        regrid_dirpath::String,
+        comms_ctx::ClimaComms.AbstractCommsContext;
         α_snow = FT(0.8),
         varname = "sw_alb",
-        infile_path = Bucket.bareground_albedo_dataset_path(),
+        get_infile_path::Function = Bucket.bareground_albedo_dataset_path,
     ) where {FT}
 
 Constructor for the BulkAlbedoStatic that implements a default albedo map, `comms` context, and value for `α_snow`.
 The `varname` must correspond to the name of the variable in the NetCDF file specified by `path`.
 
-The `bareground_albedo_dataset_path` artifact can be used as a default with this type.
+The `bareground_albedo_dataset_path` artifact will be used as a default with this type.
```

Comment:
maybe some clarity needed here re: path vs artifact? (is the artifact the data itself, while the path points to the data?)

Reply:
I added some info to the docstrings, and also renamed `get_infile_path` --> `get_infile` to hopefully emphasize that it downloads the data rather than only returning the filepath